mobile-core.addon

```diff
@@ -3,41 +3,30 @@
 # Var-Defaults: CONTAINER_REPO_USERNAME=USERNAME, CONTAINER_REPO_PASSWORD=PASSWORD, CONTAINER_REPO_ORG=aerogearcatalog, VER=v9.3.0, CORE_REPO=aerogear, CORE_BRANCH=master, CONSOLE_REPO=aerogear, CONSOLE_BRANCH=master
 # OpenShift-Version: >=3.9.0
 
+echo Installing oc client in vm
+echo
 ssh tce-load -wi bash.tcz
 ssh tce-load -wi coreutils.tcz
-
-ssh curl -s https://nodejs.org/dist/#{VER}/node-#{VER}-linux-x64.tar.gz > ~/#{VER}.tar.gz
-ssh cd /usr/local && sudo /usr/local/bin/tar --strip-components=1 -xzf ~/#{VER}.tar.gz
-
 ssh sudo chmod -R a+rwx /var/lib/minishift/openshift.local.config
 ssh curl -s -O -J -L https://github.com/openshift/origin/releases/download/v3.9.0/openshift-origin-client-tools-v3.9.0-191fece-linux-64bit.tar.gz
 ssh tar -xf openshift-origin-client-tools-v3.9.0-191fece-linux-64bit.tar.gz && sudo cp openshift-origin-client-tools-v3.9.0-191fece-linux-64bit/oc /usr/bin/oc
 ssh mkdir ~/.kube
 ssh cp /var/lib/minishift/openshift.local.config/master/admin.kubeconfig ~/.kube/config
 ssh oc login -u system:admin
```

Comment:
This block is questionable if its required. I think its only required for the broker install below. See below comment for a suggestion for removing both this block and below broker install block

Reply:
This is currently only required for the waits now, I believe, like here:
https://github.com/aerogear/minishift-mobilecore-addon/pull/21/files#diff-00ef84954f4f9f6ae930aa96c1cf34b5R29